src/main/java/org/openRealmOfStars/starMap/randomEvent/GoodRandomType.java

```diff
@@ -28,11 +28,6 @@ public enum GoodRandomType {
    * This signal is from another realm which player hasn't met yet.
    */
   MYSTERIOUS_SIGNAL,
-  /**
-   * Climate changes on planet to provide more food.
-   * DESERT->ARID->None->Lush Vegatation->Paradise
-   */
-  CLIMATE_CHANGE,
```

Comment:
This breaks following class:
`Open-Realms-of-Stars/src/main/java/org/openRealmOfStars/gui/mapPanel/PopupPanel.java:[165,14] an enum switch case label must be the unqualified name of an enumeration constant`